### ISSUE TITLE
Why does it need permission to clear recent history?

### ISSUE THREAD

**Issue by Mr-Personality** [mrdokenny/Cookie-AutoDelete#476]
I really wanted to install this add-on, but it said it needs permission to "clear recent history". That makes me very nervous. I don't want any of my history to be cleared. There's no explanation about this on the add-on page, or in the FAQ.

Why does the add-on want permission to clear recent history?

**Comment by ke-d** [comment]
- Access your data for all websites: CAD requests this to delete cookies for all sites
- Clear recent browsing history, cookies and related data: **Ability to delete Localstorage.**
- Display notifications to you: Self explanatory.
- Read and modify privacy settings: Automatic detection of First Party Isolation support. Although the Privacy permission gives CAD "read and modify" permissions, NO MODIFICATIONS are done to your privacy settings, CAD only reads the setting of First Party Isolation. The privacy permission unfortunately cannot be given granular control, so this will stay for now.
- Access browser tabs: CAD uses this permission to display the site hostname in the popup for easy whitelisting/greylisting, displaying the amount of cookies for that site, and which rules apply to that site.

Basically it is a misleading name to use this API to clear Localstorage:
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/browsingData
